### PR TITLE
[DRAFT] Mkdocs integration

### PR DESCRIPTION
**Pull request by omri374**

### CODE REVIEW

**Comment by fnocera** [comment]
@omri374 and @TessFerrandez can we set up time to discuss and talk through this?

**Comment by omri374** [comment]
@fnocera and @TessFerrandez yes with pleasure.

File: docs/CSE.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Microsoft CSE (Commercial Software Engineering)
+# Who Are We
```
Comment (fnocera) [inline docs/CSE.md:1]: Nit: "Who We Are" might have a better ring to it.

**Comment by omri374** [comment]
Closing this PR and opening a new one

**Pull request closed without merging**